_posts/2019-04-01-ogp-engine-development.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "OGP Engine Development"
+categories: ["Missouri State University"]
+tags: ["csc590", "game engine", "simulation", "omega gaming project"]
+author: "Gage Coates"
+include_mathjax: false
+---
+# Overview
```

Comment:
With Markdown, it is usually safer to put blank lines before and after headings.